pkg/cache/cache_test.go

```diff
@@ -1304,55 +1304,44 @@ func TestCacheQueueOperations(t *testing.T) {
 	}
 	cases := map[string]struct {
 		ops             []func(context.Context, client.Client, *Cache) error
-		wantQueueCounts map[string]map[string]int
+		wantQueueCounts map[string]int32
 	}{
 		"insert cqs, queues, workloads": {
 			ops: []func(ctx context.Context, cl client.Client, cache *Cache) error{
 				insertAllClusterQueues,
 				insertAllQueues,
 				insertAllWorkloads,
 			},
-			wantQueueCounts: map[string]map[string]int{
-				"foo": {
-					"ns1/alpha": 1,
-					"ns2/beta":  2,
-				},
-				"bar": {
-					"ns1/gamma": 1,
-				},
+			wantQueueCounts: map[string]int32{
+				"foo/ns1/alpha": 1,
```

Comment:
I think it would have been fine to just have "ns1/alpha"

Reply:
You are correct.  Added.